_install-and-configure/install-dashboards/tls.md

```diff
@@ -32,7 +34,6 @@ opensearch.requestHeadersAllowlist: [ authorization,securitytenant ]
 server.ssl.enabled: true
 server.ssl.certificate: /usr/share/opensearch-dashboards/config/client-cert.pem
 server.ssl.key: /usr/share/opensearch-dashboards/config/client-cert-key.pem
-server.ssl.certificateAuthorities: [ "/usr/share/opensearch-dashboards/config/root-ca.pem", "/usr/share/opensearch-dashboards/config/intermediate-ca.pem" ]
```

Comment:
I believe both this and the setting below are currently supported. Not sure why both are implemented but I in the meantime we should probably keep both documented.

Reply:
opensearch.ssl.certificateAuthorities - CA who signed the server certificate used in OpenSearch cluster
server.ssl.certificateAuthorities - CA who signed the client certificate if mTLS is enabled for Dashboard